### DISCUSSION TITLE
pulizia terminale

### DISCUSSION THREAD

**Discussion by ProssimoCentauro** [fexh10/aw-cli#12] [Ideas]
ciao, ho fatto il fork alla repo e volevo chidere se potevo fare una pull request dove ho aggiunto che se l'utente decide di uscire dal programma, quando ad esempio scrive exit o preme 'e' nella scelta delle opzioni, il terminale viene pulito con il comando clear. Ho importato subprocess per fare questo ed ho modificato solo run.py. Sarebbe un onore per me poter fare una pull request su questo progetto perchè è veramente utile e velocissimo! fammi sapere.

**Comment by fexh10** [comment]
Ciao! Guarda proprio oggi ho fatto uscire una versiona nuova ([v.2.0](https://github.com/fexh10/aw-cli/releases/tag/v2.0)) e mi sa che la tua fork è su quella vecchia. 

Il comando `clear` è già integrato nel programma, senza dover usare subprocess: inserendo `my_print("",end="", cls=True)` lo schermo viene pulito.

Le pull request sono sempre ben accette nel caso tu le voglia fare!

**Reply by ProssimoCentauro** [reply]
ammazza che sfiga volevo contribuire ahahah, comunque va bene grazie scaricherò la versione nuova

**Reply by ProssimoCentauro** [reply]
guarda ho provato a scaricare la versione 2.0 ma il terminale non viene pulito, sbaglio qualcosa?
![Screenshot from 2024-11-22 19-45-30](https://github.com/user-attachments/assets/15618eff-9e5f-4b02-81d4-f8e777c45587)
![Screenshot from 2024-11-22 19-45-46](https://github.com/user-attachments/assets/abc04faf-0ea0-4833-8edb-fc3ed8b20780)

**Comment by fexh10** [comment]
No è normale, quello che intendevo è che non è stata messa la pulizia del terminale quando si chiude il programma, ma per farlo basta aggiungere `my_print("",end="", cls=True)` invece che importare subprocess

**Reply by ProssimoCentauro** [reply]
aah scusa non avevo capito. Allora proverò a farlo e fare una pull request se per te va bene
